diff --git a/src/lib/Hydra/Plugin/SlackNotification.pm b/src/lib/Hydra/Plugin/SlackNotification.pm
index 9e576c0b..47078e85 100644
--- a/src/lib/Hydra/Plugin/SlackNotification.pm
+++ b/src/lib/Hydra/Plugin/SlackNotification.pm
@@ -139,10 +139,7 @@ sub buildFinished {
             $topbuild->buildstatus == 4 ? "warning" :
             "danger";
 
-        my $text = "";
-        $text .= "Job <$baseurl/job/${\$topbuild->jobset->get_column('project')}/${\$topbuild->jobset->get_column('name')}/${\$topbuild->get_column('job')}|${\showJobName($topbuild)}>";
-        $text .= " (and ${\scalar @deps} others)" if scalar @deps > 0;
-        $text .= ": <$baseurl/build/${\$topbuild->id}|" . showStatus($topbuild) . ">". " in " . renderDuration($topbuild);
+        my $text = "in " . renderDuration($topbuild);
 
         if (scalar keys %{$authors} > 0) {
             # FIXME: escaping
@@ -160,7 +157,7 @@ sub buildFinished {
             text => $text,
             thumb_url => $img,
             color => $color,
-            title => "Job " . showJobName($topbuild) . " build number " . $topbuild->id,
+            title => "Jobset " . $topbuild->jobset->get_column("name") . ": " . showStatus($topbuild),
             title_link => "$baseurl/build/${\$topbuild->id}"
           }]
         };
